appointment-svc/controllers/appointment-controller.ts

```diff
@@ -176,6 +184,26 @@ class AppoinmentController {
     }
   }
 
+  async deleteMany(req: Request, res: Response, next: NextFunction) {
+    const ignoredIds = req.body
+    const condition = {
+      where: {
+        id: ignoredIds
+      }
+    }
+    const ignoredAppointments = await AppointmentRepository.getMany(condition)
+
+    try {
+      ignoredAppointments.forEach(async (appointment) => {
+        await AppointmentRepository.deleteOne(appointment.id)
+      })
```

Comment:
Instead of first fetching all the appointments and then deleting all appointments 1 by 1, we can just create a new database call with deleteMany and pass a list of IDs.